main.tf

```diff
@@ -144,6 +144,7 @@ resource "aws_cloudfront_distribution" "default" {
 
     forwarded_values {
       query_string = "${var.forward_query_string}"
+      headers      = ["Access-Control-Request-Headers", "Access-Control-Request-Method", "Origin"]
```

Comment:
Should we make this a `variable` with a default of `["Access-Control-Request-Headers", "Access-Control-Request-Method", "Origin"]`?

Reply:
Although they mention only the three headers for S3 origin

```
If your origin is an S3 bucket, you typically must configure your distribution to forward and whitelist the following headers to Amazon S3:

Access-Control-Request-Headers
Access-Control-Request-Method
Origin
```

It's a good idea to move it to a variable so it could be changed if needed.